src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -336,15 +327,11 @@ def _cc_send(self, msg: MessageType, raw: bool = False, **kwargs) -> None:
         log.internal_debug(f"{self} sent CAN Message: {can_msg}, data: {_data}")
 
     def _cc_receive(
-        self, timeout: float = 0.0001, raw: bool = False
+        self, timeout: float = 0.0001
```

Comment:
you break the xcp adapter

Reply:
Sorry, I don't know see what you are talking about here

Reply:
I think that he meant that with this PR you will break additional code in our plugins where calls are made to cc_send/receive with the raw argument, e.g. the xcp_adapter